docs/docs-content/clusters/edge/upgrade-behavior.md

```diff
@@ -0,0 +1,79 @@
+
+### Kubernetes Layer
+
+<Tabs>
+<TabItem label="PXK-E" value="pxk-e">
+| Repave | Reboot | Service Restart | No-op |
+|--------|--------|-----------------|-------|
+| None. | <ul> <li>`stages.rootfs.*`</li> <li>`stages.initramfs.*`</li> <li>`stages.boot.*`</li> <li>`stages.fs.*`</li> <li>`stages.network.*`</li> <li>`stages.after-install.*`</li> <li>`stages.after-install-chroot.*`</li> <li>`stages.after-upgrade.*`</li> <li>`stages.after-upgrade-chroot.*`</li> <li>`stages.after-reset.*`</li> <li>`stages.after-reset-chroot.*`</li> <li>`stages.before-install.*`</li> <li>`stages.before-upgrade.*`</li> <li>`stages.before-reset.*`</li> </ul> | <li> `cluster.*` </li> | <li> `clientConfig.*` </li> <li>`pack.*` </li><li> Any future new keys</li> |
+
+</TabItem>
+
+<TabItem label="K3s" value="k3s">
+| Repave | Reboot | Service Restart | No-op |
+|--------|--------|-----------------|-------|
+| None. | <ul> <li>`stages.rootfs.*`</li> <li>`stages.initramfs.*`</li> <li>`stages.boot.*`</li> <li>`stages.fs.*`</li> <li>`stages.network.*`</li> <li>`stages.after-install.*`</li> <li>`stages.after-install-chroot.*`</li> <li>`stages.after-upgrade.*`</li> <li>`stages.after-upgrade-chroot.*`</li> <li>`stages.after-reset.*`</li> <li>`stages.after-reset-chroot.*`</li> <li>`stages.before-install.*`</li> <li>`stages.before-upgrade.*`</li> <li>`stages.before-reset.*`</li> </ul> | <li> `cluster.*` </li> | <li> `clientConfig.*` </li> <li>`pack.*` </li><li> Any future new keys</li> |
+
+</TabItem>
+
+<TabItem label="RKE2" value="rke2">
+| Repave | Reboot | Service Restart | No-op |
+|--------|--------|-----------------|-------|
+| None. | <ul> <li>`stages.rootfs.*`</li> <li>`stages.initramfs.*`</li> <li>`stages.boot.*`</li> <li>`stages.fs.*`</li> <li>`stages.network.*`</li> <li>`stages.after-install.*`</li> <li>`stages.after-install-chroot.*`</li> <li>`stages.after-upgrade.*`</li> <li>`stages.after-upgrade-chroot.*`</li> <li>`stages.after-reset.*`</li> <li>`stages.after-reset-chroot.*`</li> <li>`stages.before-install.*`</li> <li>`stages.before-upgrade.*`</li> <li>`stages.before-reset.*`</li> </ul> | <li> `cluster.*` </li> | <li> `clientConfig.*` </li> <li>`pack.*` </li><li> Any future new keys</li> |
+
+</TabItem>
+
+</Tabs>
+
+## Network Layer
+
+Changes made to the same Container Network Interface (CNI) pack typically do not result in cluster repave or reboot, and
+can be applied by restarting the related networking services.
+
+:::warning
+
+Do not change to a different CNI pack after provisioning a cluster. You can make changes to the existing CNI pack, but
+if you want to use a different CNI pack altogether, we recommend you re-create another cluster.
```

Comment:
re-create another cluster or create another cluster? or re-create the existing cluster? :B